docs/security/authentication/index.asciidoc

```diff
@@ -44,6 +45,31 @@ The token authentication provider can be used in conjunction with the basic auth
 xpack.security.authc.providers: [token, basic]
 --------------------------------------------------------------------------------
 
+[[pki-authentication]]
```

Comment:
__note:__ I'll double check all the links once ES documentation PR I reference to lands.

Reply:
We'll want to ensure the doc links work before merging this PR. There's a step that is ran by the docs CI which will start failing their build if the links don't work 😬

Reply:
Sure thing, there was no plan to merge this before ES docs change reaches upstream master 😛